Composite/Plugins/Functions/FunctionProviders/CodeBasedFunctionProvider/CodeBasedFunctionEntityToken.cs

```diff
@@ -0,0 +1,31 @@
+using Composite.C1Console.Security;
+using Composite.Functions;
+
+namespace Composite.Plugins.Functions.FunctionProviders.CodeBasedFunctionProvider
+{
+    [SecurityAncestorProvider(typeof(StandardFunctionSecurityAncestorProvider))]
+    internal class CodeBasedFunctionEntityToken : EntityToken
+    {
+        internal CodeBasedFunctionEntityToken(CodeBasedFunction function)
+        {
+            Id = string.Concat(function.Namespace, ".", function.Name);
```

Comment:
To mach the existing style $"{function.Namespace}.{function.Name}"

Reply:
Updated